wagtail/admin/templates/wagtailadmin/pages/listing/_page_title_choose.html

```diff
@@ -12,6 +12,9 @@
     {% else %}
         {{ page.get_admin_display_title }}
     {% endif %}
+    {% if show_locale_labels and parent_page.is_root and not page.is_root %}
```

Comment:
Can we reduce `parent_page.is_root and not page.is_root` to `page.depth == 2` perhaps? That'll avoid the need to pass `parent_page`, which means one less piece of spaghetti to untangle when we update #7780 :-)

Reply:
Done in ecfffaa